### DISCUSSION TITLE
Grief Prevention compatibility

### DISCUSSION THREAD

**Discussion by TragheitSocke** [Dans-Plugins/Medieval-Factions#1751] [Q&A] [answered]
Is the grief prevention plugin compatible with Medieval Plugins? Because although there are factions and shared buildings but, the members of my faction still want their properties with grief prevention.

**Comment by dmccoystephenson** [comment, marked as answer]
Hey @TragheitSocke,

I'm not sure which plugin you're referring to, but we haven't done any integration testing with other grief prevention plugins so I'm not sure, you should let me know if you ran into any compatibility issues!

Signed,
Daniel
